### ISSUE TITLE
Problems building Spilo image

### ISSUE THREAD

**Issue by sasadangelo** [zalando/spilo#616]
Hi all,

Until this my commit https://github.com/zalando/spilo/commit/9221a4a8414d90f40fa642f515a3e7fce22691d7 on July 21th I was able to build spilo using the command:
```
docker build . --tag spilo:latest
```
on my Mac.

Now I got the following problem:
```
  building 'google_crc32c._crc32c' extension
  creating build/temp.linux-x86_64-3.6
  creating build/temp.linux-x86_64-3.6/src
  creating build/temp.linux-x86_64-3.6/src/google_crc32c
  x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -I/usr/include/python3.6m -c src/google_crc32c/_crc32c.c -o build/temp.linux-x86_64-3.6/src/google_crc32c/_crc32c.o
  unable to execute 'x86_64-linux-gnu-gcc': No such file or directory
  error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
  
  ----------------------------------------
  Failed building wheel for google-crc32c
  Running setup.py clean for google-crc32c
```

then appear the following problem just below the text above:
```
Installing collected packages: filechunkio, google-auth, protobuf, googleapis-common-protos, google-api-core, google-cloud-core, google-crc32c, google-resumable-media, google-cloud-storage, wal-e, pg-view
  Running setup.py install for google-crc32c: started
    Running setup.py install for google-crc32c: finished with status 'error'
    Complete output from command /usr/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-jks9v07c/google-crc32c/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-hezm3t5_-record/install-record.txt --single-version-externally-managed --compile:
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-3.6
    creating build/lib.linux-x86_64-3.6/google_crc32c
    copying src/google_crc32c/_checksum.py -> build/lib.linux-x86_64-3.6/google_crc32c
    copying src/google_crc32c/python.py -> build/lib.linux-x86_64-3.6/google_crc32c
    copying src/google_crc32c/__init__.py -> build/lib.linux-x86_64-3.6/google_crc32c
    copying src/google_crc32c/__config__.py -> build/lib.linux-x86_64-3.6/google_crc32c
    copying src/google_crc32c/cext.py -> build/lib.linux-x86_64-3.6/google_crc32c
    running build_ext
    building 'google_crc32c._crc32c' extension
    creating build/temp.linux-x86_64-3.6
    creating build/temp.linux-x86_64-3.6/src
    creating build/temp.linux-x86_64-3.6/src/google_crc32c
    x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -I/usr/include/python3.6m -c src/google_crc32c/_crc32c.c -o build/temp.linux-x86_64-3.6/src/google_crc32c/_crc32c.o
    unable to execute 'x86_64-linux-gnu-gcc': No such file or directory
    error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
```

and just after this are the last lines:
```
Command "/usr/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-jks9v07c/google-crc32c/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-hezm3t5_-record/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/pip-build-jks9v07c/google-crc32c/
The command '/bin/sh -c export DEBIAN_FRONTEND=noninteractive     && set -ex     && BUILD_PACKAGES="python3-pip python3-wheel python3-dev git patchutils binutils"     && apt-get update     && apt-cache depends patroni             | sed -n -e 's/.* Depends: \(python3-.\+\)$/\1/p'             | grep -Ev '^python3-(sphinx|etcd|consul|kazoo|kubernetes)'             | xargs apt-get install -y ${BUILD_PACKAGES}                         python3-pystache python3-requests     && pip3 install setuptools     && if [ "$DEMO" != "true" ]; then         EXTRAS=",etcd,consul,zookeeper,aws"         && DISTRIB_RELEASE=$(sed -n 's/DISTRIB_RELEASE=//p' /etc/lsb-release)         && curl -sL https://github.com/wal-g/wal-g/releases/download/$WALG_VERSION/wal-g-pg-ubuntu-$DISTRIB_RELEASE-amd64.tar.gz                 | tar -C /usr/local/bin -xz         && mv /usr/local/bin/wal-g-pg-ubuntu-$DISTRIB_RELEASE-amd64 /usr/local/bin/wal-g         && strip /usr/local/bin/wal-g         && apt-get install -y python3-etcd python3-consul python3-kazoo python3-meld3                         python3-boto python3-gevent python3-greenlet python3-cachetools                         python3-rsa python3-pyasn1-modules python3-swiftclient         && find /usr/share/python-babel-localedata/locale-data -type f ! -name 'en_US*.dat' -delete         && pip3 install filechunkio wal-e[aws,google,swift]==$WALE_VERSION                 'git+https://github.com/zalando/pg_view.git@master#egg=pg-view'         && curl -sL https://github.com/wal-e/wal-e/commit/485d834a18c9b0d97115d95f89e16bdc564e9a18.diff                 | patch -d /usr/local/lib/python3.6/dist-packages/wal_e -Rp2         && sed -i 's/^\(    for i in range(0,\) num_retries):.*/\1 100):/g'                     /usr/lib/python3/dist-packages/boto/utils.py;     fi     && pip3 install patroni[kubernetes$EXTRAS]==$PATRONIVERSION     && for d in /usr/local/lib/python3.6 /usr/lib/python3; do         cd $d/dist-packages         && find . -type d -name tests | xargs rm -fr         && find . -type f -name 'test_*.py*' -delete;     done     && find . -type f -name 'unittest_*.py*' -delete     && find . -type f -name '*_test.py' -delete     && find . -type f -name '*_test.cpython*.pyc' -delete     && apt-get purge -y ${BUILD_PACKAGES}     && apt-get autoremove -y     && apt-get clean     && rm -rf /var/lib/apt/lists/*             /var/cache/debconf/*             /root/.cache             /usr/share/doc             /usr/share/man             /usr/share/locale/??             /usr/share/locale/??_??             /usr/share/info     && find /var/log -type f -exec truncate --size 0 {} \;' returned a non-zero code: 1
```

It seems that it is unable to build google_crc32. Looking at the commit history it seems that the only feature inserted is timescaledb 2.4.0 and I don't know if it could be related in some way to this problem.

Anyone can help

**Comment by CyberDem0n** [comment]
Please use the master branch.

**Comment by sasadangelo** [comment]
I don't understand your answer. I am using master branch:
```
salvatores-MBP:spilo sasadangelo$ git status
On branch master
Your branch is up to date with 'origin/master'.
```

Why are you saying I am not using it.

**Comment by sasadangelo** [comment]
Oh just saw there is a new commit related to google-crc, are you referring to this?
https://github.com/sasadangelo/spilo/commit/62d5293df82be487eb14004c77a8a8dd9891655b

**Comment by CyberDem0n** [comment]
Yes, this commit is included in the master branch and fixes build.

**Comment by sasadangelo** [comment]
Ok thx, really appreciate the quick reply and fix. Thank you very much.
